docs/source/exporters/onnx/package_reference/configuration.mdx

```diff
@@ -0,0 +1,112 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Configuration
+
+Exporting a model to ONNX involves specifying:
+1. The input names
+2. The output names
+3. The dynamic axes: basically which dimensions in the inputs will be dynamic (the other will be considered static)
+4. Dummy inputs used for tracing the model, which is needed in PyTorch to records the graph and convert it to ONNX
+
+This data being model and task dependent, it is represented as configuration classes, each architecture being associated with
```

Comment:
```suggestion
Since this data depends on the choice of model and task, we represent it in terms of _configuration classes_. Each configuration class is associated with a specific model architecture, and follows the naming convention `ArchitectureNameOnnxConfig`.
```